Makefile

```diff
@@ -5,6 +5,7 @@ SUPPORTED_ARCH=amd64 arm64
 install:
 	go mod download
 	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.55.1
```

Comment:
move this to `make deps` or `make install-dependencies`

keep `make install` for `go install .`

those are two different things